pkg/treeoperator/constants/constant.go

```diff
@@ -3,4 +3,9 @@ package constants
 const (
 	InitControllerName = "virtual-cluster-init-controller"
 	JoinControllerName = "virtual-cluster-join-controller"
+
+	DefauleImageRepositoryEnv     = "IMAGE_REPOSITIRY"
+	DefauleImageVersionEnv        = "IMAGE_VERSION"
+	VirtualClusterStatusCompleted = "Completed"
+	VirtualClusterSFinalizerName  = "kosmos.io/virtual-cluster-finalizer"
```

Comment:
There is an extra character 'S' in the word ｜ VirtualClusterFinalizerName

Reply:
fix it